### ISSUE TITLE
Invalid aria-labelledby Usage on Div Without Role in React Flow

### ISSUE THREAD

**Issue by vasifrmn** [xyflow/xyflow#5351]
### What platform were you using when you found the bug?

- React Flow / Svelte Flow version: 11.11.3
- Browser and version: Google Chrome V 137.0.7151.104
- OS and version: macos sequoia 15.4.1

### Describe the Bug

An accessibility issue is triggered when using React Flow. The following warning appears:
**The ARIA attributes "aria-labelledby" are not valid for the element <div> with implicit ARIA role "generic"**
It seems that React Flow is applying aria-labelledby to a <div> without assigning a valid ARIA role, which causes accessibility violations flagged by tools like axe-core and Lighthouse.

### Steps to reproduce the bug or issue

1. Go to a page where React Flow is rendered.
2. Click on any interactive flow element.
3. Open browser DevTools → Accessibility or run an axe accessibility scan Im used IBM equal access accessibility checker.
4. See the error: aria-labelledby applied to <div> with no valid role.

### Expected behavior

The <div> element should:
Either have a valid ARIA role (like region or group) when using aria-labelledby.
Or not have the aria-labelledby attribute if no appropriate role is assigned.

### Screenshots or Videos

<img width="692" alt="Image" src="https://github.com/user-attachments/assets/0624ddaf-6e05-4ed1-a941-5e1955afe79b" />

<img width="471" alt="Image" src="https://github.com/user-attachments/assets/01827831-4ba7-4060-8862-198956e698ab" />

**Comment by moklick** [comment]
Are you using the latest version of React Flow (12.7.x) ? Are you sure that this "aria-labelledby" div comes from React Flow?

**Comment by vasifrmn** [comment]
No, I’m currently using React Flow version 11.11.3, which is not the latest. Also, I have confirmed that this aria-labelledby div is coming from React Flow

**Comment by moklick** [comment]
Where do we use the class name "tooltip"? I don't think this comes from React Flow.. however please update to the latest version, since v11 is not supported anymore.
